src/main/resources/static/mini-manage-api-stubs.yml

```diff
@@ -507,6 +507,35 @@ paths:
         500:
           $ref: '#/components/responses/500Response'
       x-codegen-request-body-name: body
+  /person/search:
```

Comment:
Can this be `/people/search` in the spirit of RESTfulness?

Reply:
Will rectify this in a follow on PR once this and https://github.com/ministryofjustice/approved-premises-api/pull/63 are merged